### DISCUSSION TITLE
New Update?

### DISCUSSION THREAD

**Discussion by AwesomeSavage** [Darthsae/TCreator#6] [Q&A] [answered]
Is this going to be updated with the promised NPC Creation function any time soon?

**Comment by Darthsae** [comment, marked as answer]
Yes, however due to python updates I'm having to rewrite the whole thing, so it's a rather slow process. A library that was a dependency for the ai part of the NPC creation no longer supports the recent python version.

**Reply by AwesomeSavage** [reply]
Ok
